### PR TITLE
whitespace and other trivial fixes

### PR DESCRIPTION
**Pull request by jku**

### CODE REVIEW

File: p2p/information-element.h [hunk 1]
```diff
@@ -90,34 +90,34 @@ typedef struct DeviceinformationBits2_{
...
-typedef struct AssociatedBSSIDSubelement_ {
+struct __attribute__ ((packed)) AssociatedBSSIDSubelement {
     uint8_t id;
     uint16_t length;
     uint8_t bssid[6];
-} __attribute__ ((packed)) AssociatedBSSIDSubelement;
+};
 
-typedef struct CoupledSinkStatus_ {
+struct __attribute__ ((packed)) CoupledSinkStatus {
```
Comment (pozdnyakov) [inline p2p/information-element.h]: would be nice to have a macro containing the attribute

**Comment by pozdnyakov** [comment]
lgtm

File: p2p/information-element.h (hunk 1 above)
Comment (jku) [inline p2p/information-element.h]: What's the advantage of a macro? If someone went through the trouble of making that macro platform  (compiler-) independent, then I could see the point but I'm not going to even try that blindly (Why support MSVC if connman isn't used in Windows)...

Comment (pozdnyakov) [inline p2p/information-element.h]: just readability and typo protection, but I don't insist :)

**Pull request merged**